packages/material-ui/src/IconButton/IconButton.js

```diff
@@ -170,6 +163,13 @@ IconButton.propTypes = {
    * If `true`, the button will be disabled.
    */
   disabled: PropTypes.bool,
+  /**
+   * If given, uses a negative margin to counteract the padding on one
+   * side (this is often helpful for aligning the left or right
+   * side of the icon with content above or below, without ruining the border
+   * size and shape).
+   */
+  edge: PropTypes.oneOf(['left', 'right', false]),
```

Comment:
What do you think of `start | end`? It scales to RTL, you know, like this CSS property:
```ts
align-items?: 'flex-start' | 'flex-end' | 'center' | 'baseline'
```

Reply:
Huh, does `Toolbar` automatically reverse its content in RTL mode?  If so then maybe you're right that we should reverse the margin applied by `edge` in RTL mode.

Reply:
> Does Toolbar automatically reverse its content in RTL mode?

Yes, it does, you can try it on https://next.material-ui.com/. I'm trying to understand what would be more idiomatic to our users. I believe very few of them care about RTL. Hence my question, it's not obvious.

Reply:
Okay, I will change the enum to `start | end` then

Reply:
Seems to me there's enough precedent for users understanding `start | end`, even though it's a bit less obvious than `left | right` for `InputAdornment` and so forth

Reply:
@oliviertassinari oh, so looks like a JSS plugin is used to automatically flip all left and right properties in RTL mode and I don't need to check for RTL mode in my code changes -- am I understanding correctly?

Reply:
yes

Reply:
`start | end` 👍